karrot/subscriptions/tasks.py

```diff
@@ -119,14 +121,63 @@ def notify_message_push_subscribers_with_language(message, subscriptions, langua
     else:
         click_action = frontend_urls.conversation_url(conversation, message.author)
 
-    fcm_options = {
-        'message_title': message_title,
-        'message_body': Truncator(message.content).chars(num=1000),
-        # this causes each notification for a given conversation to replace previous notifications
-        # fancier would be to make the new notifications show a summary not just the latest message
-        'tag': 'conversation:{}'.format(conversation.id)
-    }
+    notify_subscribers_by_device(
+        subscriptions,
+        click_action=click_action,
+        fcm_options={
+            'message_title': message_title,
+            'message_body': Truncator(message.content).chars(num=1000),
+            # this causes each notification for a given conversation to replace previous notifications
+            # fancier would be to make the new notifications show a summary not just the latest message
+            'tag': 'conversation:{}'.format(conversation.id),
+        }
+    )
+
+
+@db_task()
+def notify_new_offer_push_subscribers(offer):
+
+    users = offer.group.members.filter(
+        groupmembership__in=GroupMembership.objects.active().with_notification_type(GroupNotificationType.NEW_OFFER),
+    ).exclude(
+        id__in=get_user_model().objects.unverified_or_ignored(),
```

Comment:
This is just relevant for emails, I think we can still push to users with unverified or ignored email addresses.